### PR TITLE
Anonymizer - Encrypt anonymizer

### PR DESCRIPTION
**Pull request by SharonHart**
- [x] Add Encrypt(Anonymizer)
- [x] Add UTs
- [x] Add integration tests (with regards to anonymizer)
- [x]  Add to open api documentation
- [x] Add readme documentation
- [x] V1 to V2 migration

Demo Website corresponding code here: 
https://github.com/microsoft/presidio-demo-website/pull/17

### CODE REVIEW

**Review by omri374 — APPROVED** [review]
Great

File: presidio-anonymizer/tests/anonymizers/test_encrypt.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+import pytest
+
+from presidio_anonymizer.anonymizers.encrypt import Encrypt
+from presidio_anonymizer.entities import InvalidParamException
+from presidio_anonymizer.services.aes_cipher import AESCipher
+
+
+@mock.patch.object(AESCipher, "encrypt")
+def test_when_anonymize_then_aes_encrypt_called_and_its_result_is_returned(
```
Comment (shiranr) [inline presidio-anonymizer/tests/anonymizers/test_encrypt.py:11]: Nit should be:
test_give_..._then_...
For all tests...

File: presidio-anonymizer/tests/integration/test_engine.py [hunk 2]
```diff
@@ -38,9 +43,30 @@ def test_when_anonymize_called_with_multiple_scenarios_then_expected_results_ret
...
+    analyzer_results = [RecognizerResult("PERSON", start_index, end_index, 0.8)]
+    anonymizers_config = {"PERSON": AnonymizerConfig("encrypt", {"key": key})}
+
+    actual_anonymize_result = AnonymizerEngine().anonymize(
+        text, analyzer_results, anonymizers_config
+    )
+
+    assert actual_anonymize_result[:start_index] == unencrypted_text
+    actual_encrypted_text = actual_anonymize_result[start_index:]
+    actual_decrypted_text = AESCipher.decrypt(key.encode(), actual_encrypted_text)
```
Comment (shiranr) [inline presidio-anonymizer/tests/integration/test_engine.py:71]: Make sure encrypted text is not the decrypt one...

Comment (SharonHart) [reply presidio-anonymizer/tests/integration/test_engine.py:72]: Done

File: presidio-anonymizer/tests/anonymizers/test_encrypt.py (hunk 1 above)
Comment (SharonHart) [reply presidio-anonymizer/tests/anonymizers/test_encrypt.py:11]: Done

Comment (shiranr) [reply presidio-anonymizer/tests/anonymizers/test_encrypt.py:11]: Not all test names were changed :)

**Review by shiranr — DISMISSED** [review]
Please notice not all test names were changed.

**Review by shiranr — APPROVED** [review]

**Review by navalev — APPROVED** [review]

**Pull request merged**